Mathlib/Computability/TuringMachine.lean

```diff
@@ -874,88 +853,79 @@ theorem exists_enc_dec [Inhabited Γ] [Finite Γ] :
 
 variable {Λ σ : Type*}
 
-local notation "Stmt₁" => Stmt Γ Λ σ
-
-local notation "Cfg₁" => Cfg Γ Λ σ
-
 /-- The configuration state of the TM. -/
 inductive Λ'
   | normal : Λ → Λ'
-  | write : Γ → Stmt₁ → Λ'
-
-local notation "Λ'₁" => @Λ' Γ Λ σ  -- Porting note (https://github.com/leanprover-community/mathlib4/issues/10750): added this to clean up types.
+  | write : Γ → Stmt Γ Λ σ → Λ'
 
-instance [Inhabited Λ] : Inhabited Λ'₁ :=
+instance [Inhabited Λ] : Inhabited (@Λ' Γ Λ σ) :=
   ⟨Λ'.normal default⟩
 
-local notation "Stmt'₁" => Stmt Bool Λ'₁ σ
-
-local notation "Cfg'₁" => Cfg Bool Λ'₁ σ
-
 /-- Read a vector of length `n` from the tape. -/
-def readAux : ∀ n, (List.Vector Bool n → Stmt'₁) → Stmt'₁
+def readAux : ∀ n, (List.Vector Bool n → Stmt Bool (@Λ' Γ Λ σ) σ) → Stmt Bool (@Λ' Γ Λ σ) σ
   | 0, f => f Vector.nil
   | i + 1, f =>
     Stmt.branch (fun a _ ↦ a) (Stmt.move Dir.right <| readAux i fun v ↦ f (true ::ᵥ v))
       (Stmt.move Dir.right <| readAux i fun v ↦ f (false ::ᵥ v))
 
-variable {n : ℕ} (enc : Γ → List.Vector Bool n) (dec : List.Vector Bool n → Γ)
+variable (n : ℕ) (enc : Γ → List.Vector Bool n) (dec : List.Vector Bool n → Γ)
```

Comment:
Why this change? You're making `n` implicit again, one declaration later.

Reply:
It's to make `n` explicit in `move`, so we don't need to introduce separate notation for it.